src/Game/DLC/aocChampionBalladManager.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <array>
+#include <container/seadSafeArray.h>
+#include <heap/seadDisposer.h>
+
+namespace uking {
+
+enum class BlightType : u32 { Wind, Electric, Fire, Water };
+
+class ChampionBalladManager {
+    SEAD_SINGLETON_DISPOSER(ChampionBalladManager);
+    ChampionBalladManager() = default;
+    virtual ~ChampionBalladManager() = default;
+
+public:
+    void init();
+    void setBlightRematchCount(s8 count, BlightType blight_type);
+    void incrementBlightRematchCount(BlightType blight_type);
+    s8 getBlightRematchCount(BlightType blight_type);
```

Comment:
```suggestion
    s8 getBlightRematchCount(BlightType blight_type) const;
```

This member function should be marked const to make it clear that it will not modify any member variables. (This is called [const correctness](https://isocpp.org/wiki/faq/const-correctness).)